docs/docs-content/clusters/public-cloud/azure/windows.md

```diff
@@ -0,0 +1,167 @@
+
+9. Choose the **Instance type**. Once selected, the cost details will be displayed.
+
+10. For the **OS Type**, choose **Windows**.
+
+11. Select the **Managed Disk** information and its size.
+
+12. Last, click on **Confirm** to create the Windows node pool.
+
+The video below showcases the process of creating a Windows node pool within an existing AKS cluster.
+
+<br />
+
+<Video title="add-windows-node-pool" src="/videos/clusters/public-cloud/azure/add-windows-node-pool.mp4"></Video>
+
+<br />
+
+### Create an Add-on Profile with a Windows Workload
+
+After creating your Windows node pool, use the following steps to create an add-on cluster profile with a Windows
+workload.
+
+13. Follow the
+    [Add a Manifest to an Add-on Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-manifest-addon.md#add-manifest-to-add-on-profile)
+    guide to create an add-on cluster profile with a custom manifest.
+
+14. Use the manifest provided below for a sample ASP.NET application, or alternatively, use your own Windows application
+    manifest. It is essential to include in the **spec** block of the manifest the **nodeSelector: "kubernetes.io/os":
+    windows** specification. This specification is required for Kubernetes to know that the application needs to be
+    deployed on a Windows node.
+
+    ```yaml
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: win-pack
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      namespace: win-pack
+      name: sample
+      labels:
+        app: sample
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          name: sample
+          labels:
+            app: sample
+        spec:
+          nodeSelector:
+            "kubernetes.io/os": windows
+          containers:
+            - name: sample
+              image: mcr.microsoft.com/dotnet/framework/samples:aspnetapp
+              resources:
+                limits:
+                  cpu: 1
+                  memory: 800M
+              ports:
+                - containerPort: 80
+      selector:
+        matchLabels:
+          app: sample
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: sample
+      namespace: win-pack
+    spec:
+      type: LoadBalancer
+      ports:
+        - protocol: TCP
+          port: 80
+      selector:
+        app: sample
+    ```
+
+### Deploy a Windows Add-on Profile to an Existing AKS Cluster
+
+Lastly, after creating your add-on cluster profile, attach it to your AKS cluster that has the previously created
+Windows node pool. Follow the steps outlined in the
+[Attach an Add-on Profile](../../../clusters/imported-clusters/attach-add-on-profile.md#attach-an-add-on-profile) guide
+to attach your add-on cluster profile to the AKS cluster.
+
+## Validate
+
+You can validate that the Windows application was successfully deployed by accessing the service URL on the AKS cluster
+**Overview** page.
```

Comment:
Usually, we don't have an intro to validation, just list the steps. I'd remove this :)